server/routes/nl.py

```diff
@@ -305,10 +305,10 @@ def _detection(orig_query, cleaned_query) -> Detection:
   ranking_classification = model.heuristic_ranking_classification(query)
   comparison_classification = model.heuristic_comparison_classification(query)
   overview_classification = model.heuristic_overview_classification(query)
-  temporal_classification = model.query_classification("temporal", query)
+  temporal_classification = None
```

Comment:
is this removal intentional?

Reply:
Yes, we weren't ever really using it (we don't really do much with anything temporal right now). The only temporal attribution/classification we're using is called "time_delta" and that is untouched by this PR.